dizoo/metadrive/env/drive_wrapper.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Dict, Optional
+from easydict import EasyDict
+from itertools import product
+from typing import NoReturn, Optional, List
+import matplotlib.pyplot as plt
+import gym
+import copy
+import numpy as np
+from ding.envs.env.base_env import BaseEnvTimestep
+from ding.envs.common.env_element import EnvElementInfo
+from ding.torch_utils.data_helper import to_ndarray
+from dizoo.metadrive.env.drive_utils import BaseDriveEnv, deep_merge_dicts
+
+
+class DriveEnvWrapper(gym.Wrapper):
+    """
+    Environment wrapper to make ``gym.Env`` align with DI-engine definitions, so as to use utilities in DI-engine.
```

Comment:
the style of annotations refer to this [file](https://github.com/opendilab/DI-engine/blob/main/ding/model/template/q_learning.py#L77)